common/lib/xmodule/xmodule/video_module/bumper_utils.py

```diff
@@ -0,0 +1,142 @@
+"""
+Utils for video bumper
+"""
+import copy
+import json
+import pytz
+import logging
+from collections import OrderedDict
+
+from datetime import datetime, timedelta
+from django.conf import settings
+
+from .video_utils import set_query_parameter
+
+try:
+    import edxval.api as edxval_api
+except ImportError:
+    edxval_api = None
+
+log = logging.getLogger(__name__)
+
+
+def get_bumper_settings(video):
+    """
+    Get bumper settings from video instance.
+    """
+    bumper_settings = copy.deepcopy(getattr(video, 'video_bumper', {}))
+
+    # clean up /static/ prefix from bumper transcripts
```

Comment:
@auraz @mikigoyal I'm afraid I don't have any context for this change, not knowing how video transcripts work or what it means for an "instructor to type transcripts in format /static/filename.srt". Can you explain more about why this code is necessary and how transcripts for bumpers differ from normal transcripts for videos?